src/Core_Screencap/Core.ScreenshotManager.cs

```diff
@@ -168,6 +169,11 @@ private void InitializeSettings()
                 "General", "Screenshot filename Name override",
                 "",
                 new ConfigDescription("Forces the Name part of the filename to always be this instead of varying depending on the name of the current game. Use \"Koikatsu\" to get the old filename behaviour.", null, "Advanced"));
+
+            UIShotUpscale = Config.Bind(
+                "UI Render Output Resolution", "UI render resolution multiplier",
+                1,
+                new ConfigDescription("Upscales the UI screenshot output resolution from the current. Allows use of AA with screenshots, but you will need to manually hide the UI first", null, "Advanced"));
```

Comment:
Add an acceptable value range parameter to the description, something like 1 - 8 or whatever is reasonable.